packages/flutter/test/material/text_field_test.dart

```diff
@@ -3341,4 +3341,38 @@ void main() {
     expect(minOffset, 0.0);
     expect(maxOffset, 50.0);
   });
+
+  testWidgets('TextField style is merged with theme', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/23994
+
+    final ThemeData themeData = ThemeData(
+      textTheme: TextTheme(
+        subhead: TextStyle(
+          color: Colors.blue[500],
+        ),
+      ),
+    );
+
+    Widget buildFrame() {
+      return MaterialApp(
+        theme: themeData,
+        home: Material(
+          child: Center(
+            child: Builder(
+              builder: (BuildContext context) {
```

Comment:
This builder shouldn't be needed

Reply:
Whoops I think I was copying that from the issue...